invenio_communities/views/decorators.py

```diff
@@ -11,18 +11,33 @@
 from functools import wraps
 
 from flask import g
+from flask_resources import CustomJSONSerializer, JSONSerializer, UIListSchema
 
+from invenio_communities.communities.resources.UISchema import \
+    UICommunitySchema
 from invenio_communities.proxies import current_communities
 
 
-def pass_community(f):
+def pass_community(serialize=False):
     """Fetch the community record."""
 
-    @wraps(f)
-    def view(**kwargs):
-        pid_value = kwargs["pid_value"]
-        community = current_communities.service.read(id_=pid_value, identity=g.identity)
-        kwargs["community"] = community
-        return f(**kwargs)
-
-    return view
+    def decorator(f):
+        @wraps(f)
+        def view(**kwargs):
+            pid_value = kwargs["pid_value"]
+            community = current_communities.service.read(
+                id_=pid_value, identity=g.identity
+            )
+            kwargs["community"] = community
+            if serialize:
+                community_ui = CustomJSONSerializer(
+                    object_key='ui',
```

Comment:
I find it a bit complex to specify the "ui" key on each usage, I had an impression we were trying to "standarize" it across the code base. How about having a new class inherited from `MarshmallowSerializer` which assigns these 4 arguments as class attributes? 
Is it possible to have it registered with the community record in the config? ping @ntarocco what do you think?

Reply:
after re-reading the PR, I see the response serializer is already registered in the resource config, but using it here in the decorator actually prevents the config from being used, I think it would be better to use the response serializer rather than implementing it here in the decorator

Reply:
The one configured in the `config` is for REST APIs, I am not sure that you can easily reuse it here.
In any case, maybe we can indeed create a `CommunitySerializer` or similar, instantiating (or inheriting if not possible to compose) `MarshmallowSerializer` with the params here.